server/datastore/mysql/migrations/tables/20250317130944_AlterHostMDMManagedCertificatesIndex.go

```diff
@@ -0,0 +1,26 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250317130944, Down_20250317130944)
+}
+
+func Up_20250317130944(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+	ALTER TABLE host_mdm_managed_certificates
+	DROP PRIMARY KEY,
+	ADD PRIMARY KEY (host_uuid, profile_uuid, ca_name)
```

Comment:
I vaguely remember some issues with a variant of mysql (might've been percona) that enforces that all tables have a PK. I would think this is ok given that it's in the same ALTER TABLE (and is probably the only way one can change a PK with that setting) but might be worth trying out prior to release.